velox/core/QueryConfig.h

```diff
@@ -93,6 +93,8 @@ class QueryConfig {
 
   static constexpr const char* kCreateEmptyFiles = "driver.create_empty_files";
 
+  static constexpr const char* kSpillPath = "experimental.spiller-spill-path";
```

Comment:
Let's not add "experimental." prefixes to property names. It might seem like a good idea at first, but in practice we never go back and remove these, so these prefixes become confusing.